app/src/main/java/ar/com/wolox/android/training/ui/training/login/LoginFragment.java

```diff
@@ -69,6 +70,10 @@ private void initMainScreen() {
         // Hide animation and show main screen
         mContentView.setVisibility(View.VISIBLE);
         mLogoGif.setVisibility(View.GONE);
+
+        User user = getPresenter().loadCredentials();
```

Comment:
Remember the view only inform events to presenter. It could be a `onInit`. Then the presenter will order what to do to the view like `showUser(user)`. By the way, the `BasePresenter` we are extending from, already has a `onViewAttached` that is called on init, so you don't need to create your own `onInit`.